$NetBSD: patch-config.lib,v 1.1 2017/06/28 18:18:36 adam Exp $

Compile 'strip' test with CFLAGS, so -isysroot can be passed on OSX.
Correct library detection on NetBSD.
On Darwin, be able to locate iconv.h inside the SDK.

--- config.lib.orig	2017-06-13 18:35:00.000000000 +0000
+++ config.lib
@@ -2095,7 +2095,7 @@ check_strip() {
 	elif [ "$os" = "OSX" ]; then
 		# Most targets have -V in strip, to see if they exists... OSX doesn't.. so execute something
 		echo "int main(int argc, char *argv[]) { }" > strip.test.c
-		$cxx_host strip.test.c -o strip.test
+		$cc_host $CFLAGS strip.test.c -o strip.test
 		check_compiler "host strip" "strip" "$host" "$strip" "$STRIP" "strip" "strip" "3" "strip.test"
 		rm -f strip.test.c strip.test
 	else
@@ -2601,7 +2601,7 @@ detect_library() {
 		if [ -z "$res" ]; then
 			log 2 "  trying /opt/local/include/$4$5... no"
 		fi
-		if [ -z "$res" ] && [ "$os" = "NETBSD" ]; then
+		if [ -z "$res" ]; then
 			eval "$2=`ls -1 /usr/pkg/include/$4*.h 2>/dev/null | egrep \"\/$5\$\"`"
 			eval "res=\$$2"
 			if [ -z "$res" ]; then
@@ -2873,6 +2873,9 @@ detect_iconv() {
 		if [ -z "$iconv" ]; then
 			iconv=`ls -1 /usr/local/include 2>/dev/null | grep "iconv.h"`
 		fi
+		if [ -z "$iconv" ] && [ "$os" = "OSX" ]; then
+			iconv=`xcrun --show-sdk-path`/usr/include/iconv.h
+		fi
 	else
 		# Make sure it exists
 		iconv=`ls $with_iconv/include/iconv.h 2>/dev/null`
